src/oemof_tabular_plugins/wefe/facades/pv_panel.py

```diff
@@ -117,6 +117,7 @@ def build_solph_components(self):
             raise ValueError("Length mismatch between t_air and ghi profiles.")
         # calculates the temperature factor values
         pv_tf_values = []
+        # TODO document those equations
```

Comment:
@JulianBarinton - the conversion factor used here for PV panel seems faulty, do you know where this model comes from?